client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -296,20 +272,290 @@ class EmailProvidersComparison extends React.Component {
+		} );
+		// TODO: calculate whether a discount/trial applies for the current domain
+		const discount = translate( '3 months free' );
+		const logo = (
+			<Gridicon
+				className="email-providers-comparison__providers-wordpress-com-email"
+				icon="my-sites"
+			/>
+		);
+		const poweredByTitan = (
+			<img src={ poweredByTitanLogo } alt={ translate( 'Powered by Titan' ) } />
+		);
+
+		const formFields = (
+			<>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Email address' ) }
+						<FormTextInputWithAffixes
+							required
+							suffix={ `@${ domain.name }` }
+							onChange={ this.onTitanEmailChange }
+						/>
+					</FormLabel>
+				</FormFieldset>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Password' ) }
+						<FormPasswordInput required onChange={ this.onTitanPasswordChange } />
+					</FormLabel>
+				</FormFieldset>
+			</>
+		);
+
+		return (
+			<EmailProviderCard
+				providerKey="titan"
+				logo={ logo }
+				title={ translate( 'Email' ) }
+				badge={ poweredByTitan }
+				description={ translate(
+					'Easy-to-use email with incredibly powerful features. Manage your email and more on any device.'
+				) }
+				formattedPrice={ formattedPrice }
+				discount={ discount }
+				formFields={ formFields }
+				buttonLabel={ translate( 'Add Email' ) }
+				onButtonClick={ this.onAddTitanClick }
+				features={ this.getTitanFeatures() }
 			/>
 		);
 	}
 
+	onGoogleUsersChange = ( changedUsers ) => {
+		this.setState( { googleUsers: changedUsers } );
+	};
+
+	onGoogleFormReturnKeyPress = ( event ) => {
+		// Simulate an implicit submission user form :)
+		if ( event.key === 'Enter' ) {
+			this.onGoogleConfirmUsers();
+		}
+	};
+
+	onGoogleConfirmNewUsers = () => {
+		const { googleUsers } = this.state;
+
+		const usersAreValid = areAllUsersValid( googleUsers );
+		if ( ! usersAreValid ) {
+			return;
+		}
+
+		const { domain, productsList, selectedSiteSlug, shoppingCartManager } = this.props;
+		const domains = [ domain ];
+		const googleProductSlug = config.isEnabled( 'google-workspace-migration' )
+			? GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY
+			: GSUITE_BASIC_SLUG;
+
+		shoppingCartManager
+			.addProductsToCart(
+				getItemsForCart( domains, googleProductSlug, googleUsers ).map( ( item ) =>
+					fillInSingleCartItemAttributes( item, productsList )
+				)
+			)
+			.then( () => {
+				this.isMounted && page( '/checkout/' + selectedSiteSlug );
+			} );
+	};
+
+	renderStackedGoogleDetails() {
+		const { currencyCode, domain, gSuiteProduct, isGSuiteSupported, translate } = this.props;
+
+		if ( ! isGSuiteSupported ) {
+			return null;
+		}
+
+		const googleProps = this.getGoogleDetailProps();
+		// If we don't have any users, initialize the list to have 1 empty user
+		const googleUsers =
+			( this.state.googleUsers ?? [] ).length === 0
+				? newUsers( domain.name )
+				: this.state.googleUsers;
+
+		const formattedPrice = translate( '{{price/}} /user /month billed yearly', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( '%(price)s per month, billed yearly' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/updated-email-comparison-page)